### PR TITLE
feat: ✨ add new scope selection

### LINKED ISSUE #50: [FEAT] use manually scope for commit
**Is your feature request related to a problem? Please describe.**

When I make commit to another open-source project, I may use the scope but do not want to create a setting for workspace.

**Describe the solution you'd like**

image

The scope typed here can be used for the current commit.

### PR DESCRIPTION
**Pull request by yi-Xu-0100**
Add new scope and only use for the current time.
This scope will not be added in workspace `seting.json`

Close #50

### CODE REVIEW

**Comment by yi-Xu-0100** [comment]
@vivaxy I appreciate it if there are any reviews here. 😀

**Comment by vivaxy** [comment]
@yi-Xu-0100 I am busy these days. Do you like to become an official collaborator to the extension?

**Comment by yi-Xu-0100** [comment]
@vivaxy I feel very honored that you have invited me to be a collaborator, and I am also happy to be a member of the maintainer. But I am not very good at developing extensions. When I become a collaborator, I will try my best to learn how to maintain it. 😀

I tested the extension of this feature in VS Code by running a debugging task, but I just added it based on your code. If there are any errors, I will fix them as soon as possible. 😊

**Comment by yi-Xu-0100** [comment]
@vivaxy I have a litter of confusion about the publishing workflow of VS Code extensions. Whether it already has action about the publishing at the same time when making a tag or not?

**Comment by vivaxy** [comment]
@yi-Xu-0100 About publishing the extension. Currently it runs only on my computer with `yarn release` command. Maybe we can:
1. Document the standard operating procedures.
2. Try to make it automatic, i.e Github Actions.

I'd like to add you as an extension member on Microsoft VSCode extension marketplace. Please email me the email address of your account on https://marketplace.visualstudio.com/.

**Comment by yi-Xu-0100** [comment]
@vivaxy I have emailed my personal information to your contact email on your GitHub profile. I am very happy to be one of the collaborators. ❤

For the publishing workflow, I am familiar with the GitHub action and try to achieve it in the future. 😀

File: package.nls.json [hunk 1]
```diff
@@ -18,7 +18,9 @@
   "extension.sources.prompt.scope.noneItem.label": "None",
   "extension.sources.prompt.scope.noneItem.detail": "No scope.",
   "extension.sources.prompt.scope.newItem.label": "New scope",
-  "extension.sources.prompt.scope.newItem.detail": "Add a workspace scope. (You can manage scopes in workspace `settings.json`.)",
+  "extension.sources.prompt.scope.newItem.detail": "Add a new workspace scope. (You can manage scopes in workspace `settings.json`.)",
+  "extension.sources.prompt.scope.newItemWithoutSetting.label": "New scope (only use once)",
```
[the remark this answers is not in the document]
Comment (yi-Xu-0100) [reply package.nls.json:22]: Ok, I will keep it carefully. 😊

**Pull request merged**